collect_app/src/main/java/org/odk/collect/android/application/initialization/SavepointsImporter.kt

```diff
@@ -0,0 +1,75 @@
+package org.odk.collect.android.application.initialization
+
+import org.odk.collect.android.storage.StoragePathProvider
+import org.odk.collect.android.storage.StorageSubdirectory
+import org.odk.collect.android.utilities.FormsRepositoryProvider
+import org.odk.collect.android.utilities.InstancesRepositoryProvider
+import org.odk.collect.android.utilities.SavepointsRepositoryProvider
+import org.odk.collect.forms.FormsRepository
+import org.odk.collect.forms.instances.InstancesRepository
+import org.odk.collect.forms.savepoints.Savepoint
+import org.odk.collect.forms.savepoints.SavepointsRepository
+import org.odk.collect.projects.ProjectsRepository
+import org.odk.collect.settings.keys.MetaKeys
+import org.odk.collect.upgrade.Upgrade
+import java.io.File
+
+class SavepointsImporter(
+    private val projectsRepository: ProjectsRepository,
+    private val instancesRepositoryProvider: InstancesRepositoryProvider,
```

Comment:
You could probably use `ProjectDependencyProvider` here to reduce a bit of setup.